lib/src/progress-bar/ProgressBar.tsx

```diff
@@ -19,22 +16,47 @@ const DxcProgressBar = ({
   const colorsTheme = useTheme();
   const backgroundType = useContext(BackgroundColorContext);
 
+  const valueProgressBar =
+    value === null || value === undefined ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100;
+  const variant = showValue ? "determinate" : "indeterminate";
+
   return (
     <ThemeProvider theme={colorsTheme.progressBar}>
       <BackgroundProgressBar overlay={overlay}>
-        <DXCProgressBar overlay={overlay} margin={margin} backgroundType={backgroundType}>
+        <DXCProgressBar overlay={overlay} margin={margin}>
           <InfoProgressBar>
-            <ProgressBarLabel overlay={overlay} backgroundType={backgroundType}>
+            <ProgressBarLabel overlay={overlay} backgroundType={backgroundType} aria-label={label || undefined}>
               {label}
             </ProgressBarLabel>
-            <ProgressBarProgress overlay={overlay} showValue={showValue} backgroundType={backgroundType}>
-              {value === null || value === undefined ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100} %
+            <ProgressBarProgress
+              overlay={overlay}
+              showValue={showValue}
+              backgroundType={backgroundType}
+              value={valueProgressBar}
+            >
+              {valueProgressBar} %
             </ProgressBarProgress>
           </InfoProgressBar>
           <LinearProgress
-            variant={showValue ? "determinate" : "indeterminate"}
-            value={value === null || value === undefined ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100}
-          />
+            role="progressbar"
+            helperText={helperText}
+            aria-valuenow={showValue ? valueProgressBar : undefined}
```

Comment:
According to [ARIA](https://w3c.github.io/aria/#progressbar): 

> The author SHOULD supply a value for [aria-valuenow](https://w3c.github.io/aria/#aria-valuenow) unless the value is indeterminate.

You must remove `showValue` condition, `aria-valuenow` should be used always.

Reply:
I would disagree on this one, as I can understand. If the progress bar is indeterminate we should omit `aria-valuenow`, that is why I used that condition there. With value `undefined`, this attribute will disappear from the DOM.

Reply:
You're right. But this comment was related to the bug about `showValue` and indeterminate progress bar.

Reply:
Well let's keep it like this until the bug is solved.